GitUI/CommandsDialogs/FormStash.Designer.cs

```diff
@@ -260,7 +260,7 @@ private void InitializeComponent()
             // 
             Stashed.ContextMenuStrip = contextMenuStripStashedFiles;
             Stashed.Dock = DockStyle.Fill;
-            Stashed.GroupByRevision = false;
+            Stashed.GroupByRevision = true;
```

Comment:
![image](https://github.com/gitextensions/gitextensions/assets/36601201/44b8d8ab-bd6a-400e-b222-c6550a70e9bc)

Reply:
fixed
Also changed for revdiff: Show group header instead of `NoFiles` if no changes, like
![image](https://github.com/gitextensions/gitextensions/assets/6248932/e4ddaf62-73ad-4ad4-819d-412a73b2f2ef)